.github/workflows/sync-packages.yaml

```diff
@@ -32,6 +32,18 @@ jobs:
                         --path var/spack/repos/ --path-rename var/spack/repos/:repos/ \
                         --path share/spack/gitlab/cloud_pipelines/ --path-rename share/spack/gitlab/cloud_pipelines/:.ci/gitlab/ \
                         --path var/spack/spack-repo-index.yaml --path-rename var/spack/spack-repo-index.yaml:spack-repo-index.yaml \
+                        --path lib/spack/spack/test --path-rename lib/spack/spack/test:repos/spack_repo/builtin/tests/ \
+                        --path .github/workflows/unit_tests.yaml \
+                        --path pytest.ini \
+                        --path .github/workflows/audit.yaml \
+                        --path .github/workflows/ci.yaml \
+                        --path .github/workflows/prechecks.yml \
+                        --path share/spack/qa/validate_last_exit.ps1 --path-rename share/spack/qa/validate_last_exit.ps1:.ci/validate_last_exit.ps1 \
+                        --path share/spack/qa/windows_test_setup.ps1 --path-rename share/spack/qa/windows_test_setup.ps1:.ci/windows_test_setup.ps1 \
+                        --path .github/CODE_OF_CONDUCT.md --path-rename .github/CODE_OF_CONDUCT.md:spack/.github/CODE_OF_CONDUCT.md \
+                        --path .github/CONTRIBUTING.md --path-rename .github/CONTRIBUTING.md:spack/.github/CONTRIBUTING.md \
```

Comment:
I misread this, why are these going to `spack/`?

Reply:
I misread your recommendation. Let me fix that!